var/spack/repos/builtin/packages/r/package.py

```diff
@@ -210,6 +210,11 @@ def setup_dependent_build_environment(self, env, dependent_spec):
 
         r_libs_path = ':'.join(r_libs_path)
         env.set('R_LIBS', r_libs_path)
+        # R_LIBS_USER gets set to a directory in HOME/R if it is not set, such as
+        # during package installation with the --vanilla flag. Set it to null
+        # to insure that it does not point to a directory that may contain R
```

Comment:
```suggestion
        # to ensure that it does not point to a directory that may contain R
```

Reply:
Thanks; brain thinking one thing, fingers going in a different direction.